safenode/src/bin/cli/files.rs

```diff
@@ -103,7 +103,7 @@ async fn download_files(file_names_dir: PathBuf, file_api: &Files) -> Result<()>
                         println!("Successfully got file {xorname} of {} bytes!", bytes.len());
                     }
                     Err(error) => {
-                        panic!("Did not get file {xorname:?} from the network! {error}")
+                        println!("Did not get file {xorname:?} from the network! {error}")
```

Comment:
Should this also retry the Get and also above retry the store. 
Perhaps argument for just printing could not Get, but I wonder should store try again and only fail after 3 attempts or something similar.

Reply:
Actually this is fine, Error back to app and let the app decide.